Python/Product/VSInterpreters/PackageManager/CondaUtils.cs

```diff
@@ -45,5 +53,73 @@ static class CondaUtils {
             var provider = componentModel.GetService<ICondaLocatorProvider>();
             return provider?.FindLocator()?.CondaExecutablePath;
         }
+
+        /// <summary>
+        /// Determine if an environment is created/managed by conda and thus should be activated.
+        /// </summary>
+        /// <param name="prefixPath">Path to the environment.</param>
+        /// <returns><c>true</c> if it is a conda environment.</returns>
+        internal static bool IsCondaEnvironment(string prefixPath) {
+            return Directory.Exists(Path.Combine(prefixPath, "conda-meta"));
```

Comment:
Shouldn't we check if Conda environment contains python.exe?

Reply:
I'll add it.